src/Identity/test/Identity.Test/UserManagerTest.cs

```diff
@@ -657,9 +657,9 @@ public async Task ClaimMethodsFailWhenStoreNotImplemented()
         {
             var manager = MockHelpers.TestUserManager(new NoopUserStore());
             Assert.False(manager.SupportsUserClaim);
-            await Assert.ThrowsAsync<NotSupportedException>(async () => await manager.AddClaimAsync(null, null));
+            await Assert.ThrowsAsync<NotSupportedException>(async () => await manager.AddClaimsAsync(null, null));
```

Comment:
Removing the GetClaimStores is what caused the regression as those are what throw the not supported exceptions

Reply:
Yes but the store wasn't really being used in `AddClaimAsync` or `RemoveClaimAsync`. Both are just calling `GetClaimStore` without the claim store being used for anything. It's in `AddClaimsAsync` and `RemoveClaimsAsync` that the store is really used. These two methods are just passing parameters along to the overloads. So we're not changing any behavior, just throwing closing to usage than we were before.

So I've changed the methods to assert for the `NotSupportedException` on the actual methods using the store.